server/integration_tests/explore_test.py

```diff
@@ -261,7 +261,7 @@ def test_detection_multivar(self):
         'compare obesity vs. poverty',
         'show me the impact of climate change on drought',
         'how are factors like obesity, blood pressure and asthma impacted by climate change',
-        'Compare "Male population" with "Female Population"',
+        'Compare Male population with Female Population',
```

Comment:
I think this is part of the test? (Multi-var handling tries to respect the double quotes)

Should we instead replace `"` in https://github.com/datacommonsorg/website/blob/cc3c3f29d3b8b13011295896c59c7bb381f7a290/server/integration_tests/explore_test.py#L91 ?

Reply:
Done